WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsInsightsDetailsViewModel.swift

```diff
@@ -314,6 +313,41 @@ class SiteStatsInsightsDetailsViewModel: Observable {
                 var rows = [ImmuTableRow]()
                 rows.append(TotalInsightStatsRow(dataRow: createFollowerTotalInsightsRow(), statSection: .insightsFollowerTotals, siteStatsInsightsDelegate: nil))
 
+                let dotComFollowersCount = insightsStore.getAllDotComFollowers()?.dotComFollowersCount ?? 0
+                let emailFollowersCount = insightsStore.getAllEmailFollowers()?.emailFollowersCount ?? 0
+                let publicizeCount = insightsStore.getPublicizeCount()
+
+                if dotComFollowersCount > 0 || emailFollowersCount > 0 || publicizeCount > 0 {
+                let chartViewModel = StatsFollowersChartViewModel(dotComFollowersCount: dotComFollowersCount,
+                                                                  emailFollowersCount: emailFollowersCount,
+                                                                  publicizeCount: publicizeCount)
+
+                //this works and returns 3 segments correctly
+                let chartViewModel = StatsFollowersChartViewModel(dotComFollowersCount: 88,
+                                                                  emailFollowersCount: 44,
+
+                                                                  publicizeCount: 16)
+
+                //this doesn't segment correctly
+//                let chartViewModel = StatsFollowersChartViewModel(dotComFollowersCount: 632,
+//                                                                  emailFollowersCount: 1,
+//                                                                  publicizeCount: 16)
+
```

Comment:
3 different ChartViewModel's that can be tweaked / played with.  1st one is actual data.  2nd and 3rd are fixed values